app/helpers/window.js

```diff
@@ -35,7 +35,8 @@ export default function (name, options) {
             x: position[0],
             y: position[1],
             width: size[0],
-            height: size[1]
+            height: size[1],
+			maximized: win.isMaximized()
```

Comment:
It's odd as I myself prefer spaces to tabs and I setup up my VS Code as such.

Anyhow, your guess about my OS choice is right and I may test in other OSes and modify the code style if you are willing to accept this PR.